Drop unnecessary pytest-cov dependency (we don't care downstream about running coverage
reports in unit tests for users).
--- a/setup.py
+++ b/setup.py
@@ -76,7 +76,6 @@ setup_args = dict(
     extras_require={
         "test": [
             "pytest !=5.3.4",
-            "pytest-cov",
             "flaky",
             "nose",  # nose because we are still using nose streams from ipython
             "ipyparallel",
